src/DnTable/__stories__/index.stories.jsx

```diff
@@ -59,50 +66,61 @@ export const Default = () => {
-                    <DnTable.HeaderCell isMono={getIsMono()}>Time</DnTable.HeaderCell>
-                    <DnTable.HeaderCell
-                        isSorted={getIsSorted()}
-                        sortDirection={getSortDirection(DnTableSortDirections.ascend)}
-                    >
-                        User
-                    </DnTable.HeaderCell>
-                    <DnTable.HeaderCell>Role</DnTable.HeaderCell>
-                    <DnTable.HeaderCell>Cause</DnTable.HeaderCell>
+            <DnTable
+                isStriped={getIsStriped()}
+                isCards={getIsCards()}
+                isFrozen={getIsFrozen()}
+                onClick={action('table-click')}
+            >
+                <DnTable.Header onClick={action('header-click')}>
+                    <DnTable.Row>
+                        <DnTable.HeaderCell position={getPosition(DnTableDataPositions.right)}>Date</DnTable.HeaderCell>
+                        <DnTable.HeaderCell isMono={getIsMono()}>Time</DnTable.HeaderCell>
+                        <DnTable.HeaderCell
+                            isSorted={getIsSorted()}
+                            sortDirection={getSortDirection(DnTableSortDirections.ascend)}
+                        >
+                            User
+                        </DnTable.HeaderCell>
+                        <DnTable.HeaderCell>Role</DnTable.HeaderCell>
+                        <DnTable.HeaderCell>Cause</DnTable.HeaderCell>
+                    </DnTable.Row>
                 </DnTable.Header>
-                <DnTable.Body>
-                    <DnTable.HeaderCell>04/14/2017</DnTable.HeaderCell>
-                    <DnTable.BodyCell>19:34 PDT</DnTable.BodyCell>
-                    <DnTable.BodyCell>yby.jupiter</DnTable.BodyCell>
-                    <DnTable.BodyCell>Admin</DnTable.BodyCell>
-                    <DnTable.BodyCell>jira123</DnTable.BodyCell>
-                </DnTable.Body>
-                <DnTable.Body>
-                    <DnTable.HeaderCell>04/14/2017</DnTable.HeaderCell>
-                    <DnTable.BodyCell>19:34 PDT</DnTable.BodyCell>
-                    <DnTable.BodyCell>yby.jupiter</DnTable.BodyCell>
-                    <DnTable.BodyCell>Admin</DnTable.BodyCell>
-                    <DnTable.BodyCell>jira123</DnTable.BodyCell>
-                </DnTable.Body>
-                <DnTable.Body>
-                    <DnTable.HeaderCell>04/14/2017</DnTable.HeaderCell>
-                    <DnTable.BodyCell>19:34 PDT</DnTable.BodyCell>
-                    <DnTable.BodyCell>yby.jupiter</DnTable.BodyCell>
-                    <DnTable.BodyCell>Admin</DnTable.BodyCell>
-                    <DnTable.BodyCell>jira123</DnTable.BodyCell>
+                <DnTable.Body onClick={action('body-click')}>
+                    <DnTable.Row onClick={action('row-click')}>
+                        <DnTable.HeaderCell>04/14/2017</DnTable.HeaderCell>
```

Comment:
Why HeaderCell here?

Reply:
This was there prior to this PR. It is there to demo the frozen style:

https://github.com/denali-design/denali-css/blob/87cf777a4067d5c9377a85f2f05d2eaa748a7115/css/denali-components.css#L1668

Example of `<th>` in body: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th

Reply:
Interesting.  LGTM then.